### ISSUE TITLE
Custom Project Templates option not available

### ISSUE THREAD

**Issue by BrianDee67** [sameersbn/docker-gitlab#2339]
I am running a local instance of GitLab CE and want to set up a custom project template containing a standard folder structure and a few boiler plate files so new projects all follow a consistent structure.

Problem is that the **Custom Project Templates** option is not visible/available in the groups->settings->general options

How can I get access to the Custom Project Templates feature?

**Comment by kkimurak** [comment]
Yes, it is correct behavior. "Custom Project Templates" (both [group level](https://docs.gitlab.com/ee/user/group/custom_project_templates.html#custom-group-level-project-templates) and [instance level](https://docs.gitlab.com/ee/user/admin_area/custom_project_templates.html)) are available for GitLab EE, not CE that this project uses.  
P.S. A PR to add support for EE were going on #2329 (I'm not sure if it is active)
